src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -218,6 +218,56 @@ limitations under the License.
             <!-- Conformance feature [COTA & PIN] - for now optional -->
             <arg name="PINCode" type="octet_string" optional="true" />
         </command>
+        <!-- Command if not USR & PIN feature -->
+        <command source="client" code="5" name="SetPINCode" mustUseTimedInvoke="true" optional="true">
```

Comment:
Matter does not allow supporting PIN but not supporting USR.  This stuff is all Zigbee-only.  See device library spec.

Reply:
We really need to figure out a way to make this clearer in the spec. It seems like it causes a bit of confusion having stuff specified in the cluster spec but then disallowed in the device type.

Reply:
@bzbarsky-apple, thanks a lot for pointing that out.
I think that we can delete that PR, since it's not provide any additional value to Matter code.

On the one side I agree with you @ReneJosefsen that it should be in more clear way defined the app cluster spec, but on the other hand the definition do e.g. door lock device type is clear that you should support USR feature if you want to support PIN.
I think that we should propagate (docs, webinars, etc.) with information how to build the Matter product step by step from specification perspective e.g.
1. Choose device types included in the product
2. Choose necessary clusters/features etc based on required elements for specfic device type
...
...

Reply:
Yeah, not sure what is the best way to go here, it properly needs a bit of thought. But sometimes the gap from the clusters to the device types seems to be big, since it might not be obvious that the device type can overrule stuff in the cluster.